incubator/vault/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "vault.serviceAccountName" . }}
+  labels:
+    app: {{ template "vault.name" . }}
+    chart: {{ template "vault.chart" . }}
```

Comment:
This is incorrect as the `vault.chart` template doesn't exist - did you forget to add it?

Or you can use what the other vault manifests use: `{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}`

Any chance of a quick fix for this @jbialy ?

Reply:
Ah, good catch! I likely grabbed the `serviceaccount` definition from another chart, and overlooked the way the `chart` label is set!

I'll adjust it to use `{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}` as you suggested!